src/components/content/order/ServiceList.tsx

```diff
@@ -195,12 +200,25 @@ function ServiceList(): JSX.Element {
             });
     }
 
+    function migrate(record: ServiceVo): void {
```

Comment:
migration modal is still not fitting the screen fully. it is overflowing on both big and small screen.

Reply:
fixed.